demisto_sdk/commands/common/tools.py

```diff
@@ -606,18 +606,20 @@ def get_from_version(file_path):
     data_dictionary = get_yaml(file_path) if file_path.endswith('yml') else get_json(file_path)
 
     if data_dictionary:
+
         from_version = data_dictionary.get('fromversion') if 'fromversion' in data_dictionary \
-            else data_dictionary.get('fromVersion', DEFAULT_CONTENT_ITEM_FROM_VERSION)
-        if from_version == '':
-            return DEFAULT_CONTENT_ITEM_FROM_VERSION
+            else data_dictionary.get('fromVersion', '')
+
+        if not from_version:
+            return ''
 
         if not re.match(r'^\d{1,2}\.\d{1,2}\.\d{1,2}$', from_version):
             raise ValueError(f'{file_path} fromversion is invalid "{from_version}". '
                              'Should be of format: "x.x.x". for example: "4.5.0"')
 
         return from_version
 
-    return DEFAULT_CONTENT_ITEM_FROM_VERSION
+    return ''
```

Comment:
please add `logging.warning` or `logging.debug` (choose based on how much it spams the log) that the `fromversion` was not found, adding the content item name.

Reply:
done